### ISSUE TITLE
Concept : Research Table

### ISSUE THREAD

**Issue by Helkus** [Facepunch/rust#170]
Lets get some research/drafting table concept going

![image](https://cloud.githubusercontent.com/assets/4874057/7230955/00ac7e30-e740-11e4-9068-3c93262fa1df.png)

![image](https://cloud.githubusercontent.com/assets/4874057/7230971/23b64e74-e740-11e4-97f4-cc6309ca3de2.png)

magnifier thing like from kleiners lab in hl2 ? except more ghetto

https://www.google.ca/search?q=worktable+magnifying+glass&biw=3413&bih=1753&source=lnms&tbm=isch&sa=X&ei=Qv80VYa2OY7asATQ3IGwAg&ved=0CAYQ_AUoAQ&dpr=0.75#tbm=isch&q=magnifying+desk+lamp&revid=1049752415

**Comment by y0himba** [comment]
YES!  Research tables!  Will these require research kits as in legacy?  I think having a table, constructable after finding a blueprint (rare) would be more playable than finding research kits.

I see a primitive desk with rough paper, rough tools and parts scattered on it. The magnifier-were will the glass come from?  How was it blown? Instead, how about adding lenses to each end of an empty bean can, or crystals mined from the new crystals for magnifying?
